doc/source/f2py/index.rst

```diff
@@ -36,6 +36,7 @@ replace all calls to ``f2py`` mentioned in this guide with the longer version.
 
    f2py-user
    f2py-reference
+   f2py-testing
```

Comment:
Looking at the rendered docs, perhaps the best place for this is under the Reference Guide --> `numpy/doc/source/f2py/f2py-reference.rst`

Reply:
Right, I changed it. Please have a look.